bucket/fysty.json

```diff
@@ -0,0 +1,52 @@
+{
+    "version": "20220202-76f785d5",
+    "description": "A fork of the SSH/Telnet client PuTTY",
```

Comment:
We should mention something about this particular fork - what separated it from PuTTY and/or other forks of PuTTY.